forge-gui/res/adventure/Shandalar/world/items.json

```diff
@@ -1179,5 +1179,20 @@
         "Slobad's Iron Boots"
       ]
 }
+},
```

Comment:
The line before this would need to be removed, otherwise this creates invalid JSON.

Reply:
This is an artifact of how the rest of the file is formatted.  I kept the whitespace positioning in line with the other blocks.  I can assure you that the json is valid, otherwise I wouldn't have been able to test this at all.

Reply:
Got it. This is what I get for reviewing changes half asleep.